### PR TITLE
Added error handling for w2v model loading

### PR DESCRIPTION
**Pull request by AindriyaBarua**
Hello @aditeyabaral 
I was trying to build and run this project when I came across a re-occuring error:
On my console:
w2v_error
I tried on colab too:
WORD2VEC_COLAB

On investigating more on the issue, I found out that since the size of the pre-trained word2vec-google-news-300 model is around 1.5 GB, on an unstable internet, this download may fail very often, as it happened with me multiple times. So I added some error handling on that.

Previously the w2v model was loaded without a function, I have made a function for it with proper error handling, it tries to load the model, on meeting an error, it retries up to 5 times before throwing the error and terminating.

I have called the function before the user enters the Query, that is outside the main while loop, so that the model is loaded only once and can be reused every time the user wants to make new queries, hence saving us from doing memory and data consuming task multiple times.

Hope the contribution is useful, I am open to further discussion or modification.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 print("Welcome to NewsNow!")
 choice = 'y'
+w2v_model = utils.load_embedding_model()
```
Comment (aditeyabaral) [inline src/main.py]: Change the function name to `loadEmbeddingModel`

File: src/utils.py [hunk 2]
```diff
@@ -13,10 +12,28 @@
 from GoogleNews import GoogleNews
 
 stopwords = stopwords.words("english")
-# See available gensim word2vec models here: https://github.com/RaRe-Technologies/gensim-data#models
-w2v_model = api.load("word2vec-google-news-300")
 
 
+def load_embedding_model():
```
Comment (aditeyabaral) [inline src/utils.py]: Change of function name to `loadEmbeddingModel`

File: src/utils.py [hunk 3]
```diff
@@ -43,8 +60,8 @@ def getDocuments(urls):
     return articles
 
 
-def merge(documents, threshold=0.85):
-
+def merge(documents, w2v_model):
+    threshold = 0.85
```
Comment (aditeyabaral) [inline src/utils.py]: Add threshold to the function call parameters. This will allow users to tweak it as necessary.

File: src/main.py (hunk 1 above)
Comment (AindriyaBarua) [reply src/main.py]: Are you sure about this though? The naming convention for Python is : words separated by underscores as mentioned in [here](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), unlike in Java where we use camelCase.

I notice there is inconsistency in the naming convention followed in the project, let me know if you want me to change all the names to the python convention of separating by underscores, or all to camelCase.

Comment (aditeyabaral) [reply src/main.py]: Yes, but I think majority of the project uses camelCase for function headers and underscore separated lowercase for variables

Comment (aditeyabaral) [reply src/main.py]: If you can make it all uniform, go for it :)

File: src/utils.py (hunk 3 above)
Comment (AindriyaBarua) [reply src/utils.py]: @aditeyabaral  I have made the changes asked for, please do check. Please add hacktoberfest and hacktoberfest-accepted tags while merging 😋

**Pull request merged**